### DISCUSSION TITLE
Two different places for storage folder [docker]

### DISCUSSION THREAD

**Discussion by maaper** [jlesage/docker-czkawka#22] [Q&A] [answered]
Hi, is there any possibility to get access to two different folders within my server at set them up as storage?

**Comment by jlesage** [comment]
Yes, you can map any number of folders to the container.  If you use the `docker run` command to create the container, you just need to specify another `-v` parameter.

**Reply by Giant-Li** [reply]
I use QNAP and docker-composer, and I want add 2 physical disks to the storage folder. Then what shall I do?

**Reply by jlesage** [reply, marked as answer]
You can add another mapping under the `volumes:` section. For example:
```
    volumes:
      - "/path1:/storage:rw"
      - "/path2:/storage2:rw"
```
If you want both to be under `/storage`:
```
    volumes:
      - "/path1:/storage/path1:rw"
      - "/path2:/storage/path2:rw"
```

**Reply by Giant-Li** [reply]
Yeah, this works perfectly! Thank you!
